README.md

```diff
@@ -21,15 +21,14 @@ Then create a file called `addons.js` in your storybook config.
 Add following content to it:
 
 ```js
-import '@kadira/storybook/addons';
 import 'react-storybook-addon-backgrounds/register';
 ```
 
 Then write your stories like this:
 
 ```js
 import React from 'react';
-import { storiesOf } from "@kadira/storybook";
+import { storiesOf } from "@storybook/react";
 import backgrounds from "react-storybook-addon-backgrounds";
```

Comment:
Thank you for the PR mate, i just released, please let me know if it works for you?